app/server/jest.config.js

```diff
@@ -2,5 +2,6 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  testTimeout: 120000
+  testTimeout: 120000,
+  maxWorkers: 1
```

Comment:
...but I think the main issue was that the persistence tests were blitzing redis, and they were potentially running in parallel with the error tests. So limiting to sequential run here...